demo/angular-timeago.js

```diff
@@ -46,207 +46,246 @@ angular.module('yaru22.angular-timeago', []).directive('timeAgo', [
+          prefixAgo: null,
+          prefixFromNow: 'daqui a',
+          suffixAgo: 'atr\xe1s',
+          suffixFromNow: null,
+          seconds: 'menos de um minuto',
+          minute: 'cerca de um minuto',
+          minutes: '%d minutos',
+          hour: 'cerca de uma hora',
+          hours: 'cerca de %d horas',
+          day: 'um dia',
+          days: '%d dias',
+          month: 'cerca de um m\xeas',
+          months: '%d meses',
+          year: 'cerca de um ano',
+          years: '%d anos',
+          numbers: []
+        },
+        'fr_FR': {
+          prefixAgo: 'il y a',
+          prefixFromNow: null,
+          suffixAgo: null,
+          suffixFromNow: 'from now',
+          seconds: 'moins d\'une minute',
+          minute: 'environ une minute',
+          minutes: '%d minutes',
+          hour: 'environ une heure',
+          hours: 'environ %d heures',
+          day: 'un jour',
+          days: '%d jours',
+          month: 'environ un mois',
+          months: '%d mois',
+          year: 'environ un an',
+          years: '%d ans',
+          numbers: []
+        },
+        'es_LA': {
+          prefixAgo: 'hace',
+          prefixFromNow: null,
+          suffixAgo: null,
+          suffixFromNow: 'apartir de ahora',
+          seconds: 'menos de un minuto',
+          minute: 'un minuto',
+          minutes: '%d minutos',
+          hour: 'una hora',
+          hours: '%d horas',
+          day: 'un dia',
+          days: '%d dias',
+          month: 'un mes',
+          months: '%d meses',
+          year: 'un a\xf1o',
+          years: '%d a\xf1os',
+          numbers: []
+        },
+        'nl_NL': {
+          prefixAgo: null,
+          prefixFromNow: 'over',
+          suffixAgo: 'geleden',
+          suffixFromNow: 'vanaf nu',
+          seconds: 'een paar seconden',
+          minute: 'ongeveer een minuut',
+          minutes: '%d minuten',
+          hour: 'een uur',
+          hours: '%d uur',
+          day: 'een dag',
+          days: '%d dagen',
+          month: 'een maand',
+          months: '%d maanden',
+          year: 'een jaar',
+          years: '%d jaar',
+          numbers: []
+        }
       }
-    }
-  };
-  service.inWords = function (distanceMillis) {
-    var lang = document.documentElement.lang;
-    var $l = service.settings.strings[lang];
-    if (typeof $l === 'undefined') {
-      $l = service.settings.strings['en_US'];
-    }
-    var prefix = $l.prefixAgo;
-    var suffix = $l.suffixAgo;
-    if (service.settings.allowFuture) {
-      if (distanceMillis < 0) {
-        prefix = $l.prefixFromNow;
-        suffix = $l.suffixFromNow;
+    };
+    service.inWords = function (distanceMillis, fromTime, format, timezone) {
+      var fullDateAfterSeconds = parseInt(service.settings.fullDateAfterSeconds, 10);
+      if (!isNaN(fullDateAfterSeconds) && (distanceMillis >= 0 && fullDateAfterSeconds * 1000 <= distanceMillis || fullDateAfterSeconds * 1000 >= distanceMillis)) {
```

Comment:
I believe this line should be (it doesn't make sense otherwise):
if (!isNaN(fullDateAfterSeconds) && (distanceMillis >= 0 && fullDateAfterSeconds * 1000 <= distanceMillis || distanceMillis < 0 && fullDateAfterSeconds * 1000 >= distanceMillis)) {